scripts/CollatePortalJavaScriptSource.groovy

```diff
@@ -6,7 +6,7 @@
  *
  */
 
-includeTargets << grailsScript("Init")
+includeTargets << grailsScript("_GrailsInit")
```

Comment:
We should have a look at replacing this custom script with one of the standard grails plugins for minification/compression etc.  IIRC, the reason we were blocked on that in the past was because of some weird behaviour with CSS includes in IE (bit hazy - maybe someone else can elaborate?), which we no longer support anyway.